src/encoded/schemas/atac_xcor_score_quality_metric.json

```diff
@@ -0,0 +1,120 @@
+{
+    "title": "ATAC-seq cross correlation quality metric",
+    "description": "Schema for reporting ATAC-seq cross correlation quality metrics",
+    "id": "/profiles/atac_xcor_score_quality_metric.json",
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "type": "object",
+    "required": ["step_run","quality_metric_of", "award", "lab"],
+    "additionalProperties": false,
+    "identifyingProperties": ["uuid"],
+    "mixinProperties": [
+        { "$ref": "mixins.json#/schema_version" },
+        { "$ref": "quality_metric.json#/properties" },
+        { "$ref": "mixins.json#/uuid" },
+        { "$ref": "mixins.json#/submitted"},
+        { "$ref": "mixins.json#/aliases"},
+        { "$ref": "mixins.json#/attribution" },
+        { "$ref": "mixins.json#/standard_status"},
+        { "$ref": "mixins.json#/assay" }
+   ],
+    "properties": {
+        "schema_version": {
+            "default": "1"
+        },
+        "num_reads": {
+            "title": "Number of reads",
+            "type": "integer",
+            "minimum": 0
+        },
+        "est_frag_len": {
+            "title": "Estimated fragment length",
+            "type": "number",
+            "description": "Fragment length/strandshift. This is the estimated fragment length/strand shift for each dataset as estimated by strand cross-correlation analysis",
+            "minimum": 0
+        },
+        "corr_est_frag_len": {
+            "title": "Correlation at estimated fragment length",
+            "description": "Value of correlation function at estimated fragment length.",
+            "type": "number",
+            "minimum": 0,
+            "maximum": 1
+        },
+        "phantom_peak": {
+            "title": "Phantom peak",
+            "type": "integer"
+        },
+        "corr_phantom_peak": {
+            "title": "Correlation at phantom peak",
+            "type": "number",
+            "minimum": 0,
+            "maximum": 1
+        },
+        "argmin_corr": {
```

Comment:
Can argmin be negative?

Reply:
Yes, it can be in terms of schema. On some plots and according to their paper, the strand shift on x-axis can start at -100bp.